exercises/secret-handshake/introduction.md

```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+You are starting a secret coding club with some friends and friends-of-friends.
+You don't know everyone in the club, so you and your friends have decided to create a secret handshake that you can use to recognise that someone is a member.
+You don't want anyone who isn't in the know to be able to crack the code.
+
+You designed the code so that one person says a number between 1 and 31, and the other other person turns it into a series of actions.
```

Comment:
```suggestion
You've designed the code so that one person says a number between 1 and 31, and the other other person turns it into a series of actions.
```